app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -20,6 +20,8 @@ def ems_inv_to_hashes
     get_vdcs
     get_vapps
     get_vms
+    get_private_images if @options.get_private_images
```

Comment:
But yes for now the VMs under the `vapp_template` match the model for a `Template` better than a catalog_item, the `OrchestrationTemplate` should be a separate PR